armory/data/datasets.py

```diff
@@ -422,8 +422,10 @@ def _generator_from_tfds(
             download_and_prepare_kwargs=download_and_prepare_kwargs,
             shuffle_files=shuffle_files,
         )
-    except AssertionError as e:
-        if not str(e).startswith("Unrecognized instruction format: "):
+    except (AssertionError, ValueError) as e:
+        if not str(e).startswith(
+            "Unrecognized instruction format: "
+        ) and "Unrecognized split format: " not in str(e):
             raise
         log.warning(f"Caught AssertionError in TFDS load split argument: {e}")
```

Comment:
nit: this message is technically incorrect: "AssertionError or ValueError" but, :shrug:

Reply:
This is super wonky block of code that I plan to remove with the datasets update.